shell/src/main/java/alluxio/cli/docgen/ConfigurationDocGenerator.java

```diff
@@ -47,9 +46,36 @@
   private static final String CSV_FILE_DIR = "docs/_data/table/";
   private static final String YML_FILE_DIR = "docs/_data/table/en/";
   public static final String CSV_FILE_HEADER = "propertyName,defaultValue";
+  public static final String TEMP_PREFIX = "temp-";
+
+  public static final String[] CSV_FILE_NAMES = {"user-configuration.csv", "master-configuration.csv",
```

Comment:
nit: when listing a long series of strings (ex. 3 or more), i like to put them each on their own line and sort them alphabetically to be able to quickly scan for a specific file. most of the alluxio codebase does not follow this but i'm slowly making this a pattern whenever i see the opportunity